### ISSUE TITLE
selenium4 beta - firefox_profiles.py 'No such file or directory: webdriver_prefs.json'

### ISSUE THREAD

**Issue by fenchu** [SeleniumHQ/selenium#8469]
## Environment

OS:  win10 1909
Browser:  firefox
Browser version:  77.0.1
Browser Driver version:  Geckodriver 0.26.0
Language Bindings version:  Python selenium==4.0.0a6.post1

I'm trying to use a profile, I create a profile using firefox about:profiles and create a new profile.
This profile works fine from within firefox.

But when I try to load it with selenium using:
```
        elif os.environ['WEBDRIVER'] == 'Firefox':
            fp = None
            if os.environ.get('WEBDRIVER_PROFILE') and os.environ['WEBDRIVER_PROFILE']:
                if os.path.isdir(os.environ['WEBDRIVER_PROFILE']):
                    fp = webdriver.FirefoxProfile(os.environ['WEBDRIVER_PROFILE'])
            driver = webdriver.Firefox(firefox_profile=fp)
```
I get this error:
```
conftest.py:109: in wd
    driver =tools.selenium.myWebdriver(x, y)
tools\selenium.py:127: in myWebdriver
    fp = webdriver.FirefoxProfile(os.environ['WEBDRIVER_PROFILE'])
..\..\venvs\ssid-enrollment-client-test\lib\site-packages\selenium\webdriver\firefox\firefox_profile.py:58: in __init__
    with open(os.path.join(os.path.dirname(__file__),
E   FileNotFoundError: [Errno 2] No such file or directory: 'c:\\dist\\venvs\\ssid-enrollment-client-test\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver_prefs.json'
```
The firefox_profile.py under site_packages\\selenium\\webdriver\\firefox\\firefox_profile.py have this:
```
WEBDRIVER_EXT = "webdriver.xpi"
WEBDRIVER_PREFERENCES = "webdriver_prefs.json"
EXTENSION_NAME = "fxdriver@googlecode.com"


class AddonFormatError(Exception):
    """Exception for not well-formed add-on manifest files"""


class FirefoxProfile(object):
    ANONYMOUS_PROFILE_NAME = "WEBDRIVER_ANONYMOUS_PROFILE"
    DEFAULT_PREFERENCES = None

    def __init__(self, profile_directory=None):
        """
        Initialises a new instance of a Firefox Profile

        :args:
         - profile_directory: Directory of profile that you want to use. If a
           directory is passed in it will be cloned and the cloned directory
           will be used by the driver when instantiated.
           This defaults to None and will create a new
           directory when object is created.
        """
        if not FirefoxProfile.DEFAULT_PREFERENCES:
            with open(os.path.join(os.path.dirname(__file__),
                                   WEBDRIVER_PREFERENCES)) as default_prefs:
                FirefoxProfile.DEFAULT_PREFERENCES = json.load(default_prefs)
```
Was not webdriver.xpi replaced with geckodriver a long long time ago, strange this should show up in selenium4?

Thanks

**Comment by AutomatedTester** [comment]
reopening to investigate

**Comment by AutomatedTester** [comment]
looks like this was somehow missed out...

**Comment by AutomatedTester** [comment]
It looks like I broke during the bazel tweaking but now the bazel wheel code is broken

**Comment by AutomatedTester** [comment]
This was fixed in ee716a169d7dd3fae935ea21032ebbb1c8d3bea0
